app/src/main/java/bluedot/com/au/bluedotbrazeintegrationapp/BluedotGeoTriggerReceiver.kt

```diff
@@ -2,47 +2,46 @@ package bluedot.com.au.bluedotbrazeintegrationapp
 
-    override fun onZoneEntryEvent(geoTriggerEvent: GeoTriggerEvent, context: Context) {
-        println("Zone ${geoTriggerEvent.zoneInfo.name}, fence ${geoTriggerEvent.entryEvent()?.fenceName} entered at: ${Date()}")
+    override fun onZoneEntryEvent(entryEvent: GeoTriggerEvent, context: Context) {
+        println("Zone ${entryEvent.zoneInfo.name}, fence ${entryEvent.entryEvent()?.fenceName} entered at: ${Date()}")
 
         val eventProperties = BrazeProperties()
-        eventProperties.addProperty("zone_id", geoTriggerEvent.zoneInfo.id)
-        eventProperties.addProperty("zone_name", geoTriggerEvent.zoneInfo.name)
+        eventProperties.addProperty("zone_id", entryEvent.zoneInfo.id.toString())
+        eventProperties.addProperty("zone_name", entryEvent.zoneInfo.name)
         eventProperties.addProperty("latitude",
-            geoTriggerEvent.entryEvent()?.locations?.get(0)?.latitude
+            entryEvent.entryEvent()?.locations?.get(0)?.latitude
         )
-        eventProperties.addProperty("longitude", geoTriggerEvent.entryEvent()?.locations?.get(0)?.longitude)
-        eventProperties.addProperty("fence_id", geoTriggerEvent.entryEvent()?.fenceId)
-        eventProperties.addProperty("fence_name", geoTriggerEvent.entryEvent()?.fenceName)
+        eventProperties.addProperty("longitude", entryEvent.entryEvent()?.locations?.get(0)?.longitude)
+        eventProperties.addProperty("fence_id", entryEvent.entryEvent()?.fenceId.toString())
+        eventProperties.addProperty("fence_name", entryEvent.entryEvent()?.fenceName)
 
-        geoTriggerEvent.zoneInfo.customData?.forEach { data ->
+        entryEvent.zoneInfo.customData.forEach { data ->
             eventProperties.addProperty(data.key, data.value)
         }
 
         Braze.getInstance(context).logCustomEvent(customEventEntry, eventProperties)
     }
 
-    override fun onZoneExitEvent(geoTriggerEvent: GeoTriggerEvent, context: Context) {
-        println("Zone ${geoTriggerEvent.zoneInfo.name}, fence ${geoTriggerEvent.exitEvent()?.fenceName} exited at: ${Date()}")
+    override fun onZoneExitEvent(exitEvent: GeoTriggerEvent, context: Context) {
+        println("Zone ${exitEvent.zoneInfo.name}, fence ${exitEvent.exitEvent()?.fenceName} exited at: ${Date()}")
 
         val eventProperties = BrazeProperties()
-        eventProperties.addProperty("zone_id", geoTriggerEvent.zoneInfo.id)
-        eventProperties.addProperty("zone_name", geoTriggerEvent.zoneInfo.name)
-        eventProperties.addProperty("dwellTime", geoTriggerEvent.exitEvent()?.dwellTime)
-        eventProperties.addProperty("fence_id", geoTriggerEvent.exitEvent()?.fenceId)
-        eventProperties.addProperty("fence_name", geoTriggerEvent.exitEvent()?.fenceName)
+        eventProperties.addProperty("zone_id", exitEvent.zoneInfo.id.toString())
+        eventProperties.addProperty("zone_name", exitEvent.zoneInfo.name)
+        eventProperties.addProperty("dwellTime", exitEvent.exitEvent()?.dwellTime)
```

Comment:
[I] Ditto for `exitEvent.exitEvent` may look confusing.

Reply:
Same explanation as for entryEvent fixing the IDE warnings